src/list.js

```diff
@@ -0,0 +1,24 @@
+export default class Task {
+  constructor(taskEntry) {
+    this.taskEntry = taskEntry;
+    this.completed = false;
+    this.index = 0;
+  }
+
+  addTask(taskList) {
+    const myTask = new Task(this.taskEntry);
+    myTask.index = taskList.length;
+    taskList.push(myTask);
+    localStorage.setItem('taskDetail', JSON.stringify(taskList));
+  }
+
+  static delTask(index, taskList) {
+    taskList = taskList.filter((tasks, i) => index !== i);
+    localStorage.setItem('taskDetail', JSON.stringify(taskList));
+    window.location.reload();
+    taskList.forEach((taskindex, i) => {
+      taskindex.index = i;
```

Comment:
- [ ] It is a requirement that the index property is set to the value of the new array length i.e the first task has an index equal to 1, 2nd task with an index value of 2. Presently the task index value is taking the value of the default index i.e first task has an index equal to zero. Kindly fix this.